BitwardenShared/UI/Tools/Generator/Generator/GeneratorView.swift

```diff
@@ -125,7 +184,7 @@ struct GeneratorView: View {
                     store.send(.sliderEditingChanged(field: sliderField, isEditing: isEditing))
                 } onValueChanged: { newValue in
                     store.send(.sliderValueChanged(field: sliderField, value: newValue))
-                }
+                }.id("\(GuidedTourStep.step4.rawValue)")
```

Comment:
🎨 Given that `GuidedTourStep` already has an `id` which returns `rawValue`, I think it'd be better to use that directly.
```suggestion
                }.id(GuidedTourStep.step4.id)
```

Reply:
Yeah, I probably added this before I added ```id``` and forgot to switch this back. 
thanks!